is_core/rest/resource.py

```diff
@@ -281,7 +281,9 @@ def get_field_labels(self):
 
     def get_field_label(self, field_name):
         return get_field_label_from_path(
-            self.model, field_name, field_labels=self.get_field_labels()
+            self.model,
+            field_name[:-11] if field_name.endswith('___obj_name') else field_name,
+            field_labels=self.get_field_labels()
```

Comment:
problem between is-core and pyston. Pyston automatically converts _obj_name to str. But I do not want to do it in the is-core so if field name contains _obj_name which is  str function I only generates field label from prefix (eshop___obj_name, order___obj_name, etc)

Reply:
maybe use a constant to define the `___obj_name` string?

Reply:
but that might require len(OBJ_NAME) instead of 11 as well

Reply:
good point

Reply:
Triple underscore? really?

Reply:
This is not tripple underscore. The method is called _obj_name (because this is default method for all models and we didn't want use obj_name in the past). The separator is double underscore therefore the result is

prefix+__+_obj_name

Reply:
I used constants it shoud help